cmake/std/atdm/ats2/environment.sh

```diff
@@ -258,11 +255,65 @@ export F77=mpifort
 export FC=mpifort
 export F90=mpifort
 
-# Define function atdm_run_script_on_compute_node
+# Default install location
+export ATDM_CONFIG_TRIL_CMAKE_INSTALL_PREFIX_DATE_BASE_DEFAULT=/projects/atdm_devops/trilinos_installs/
+
+# System-info for what ATS-2 system we are using
+if [[ "${ATDM_CONFIG_KNOWN_HOSTNAME}" == "vortex" ]] ; then
+  export ATDM_CONFIG_ATS2_LAUNCH_NODE=vortex59
+else
+  echo "Error, the ats2 env on system '${ATDM_CONFIG_KNOWN_HOSTNAME}'"
+  return
+fi
+
+
+#
+# Define functions for running on compute nodes
+#
+
+
+function atdm_ats2_get_allocated_compute_node_name() {
+  if [[ "${LSB_HOSTS}" != "" ]] ; then
+    echo ${LSB_HOSTS} | cut -d' ' -f2
```

Comment:
Same as above.

Reply:
`ats2-cuda-10.1.243-gnu-7.3.1-spmpi-2019.06.24_static_opt` returns the empty string if you are not on a compute node.  For example:

```
[rabartl@vortex60 Trilinos (cdofa-102-ats2-fix-run-on-compute-node)]$ . cmake/std/atdm/load-env.sh cuda
Hostname 'vortex60' matches known ATDM host 'vortex' and system 'ats2'
Setting compiler and build options for build-name 'cuda'
Using ats2 compiler stack CUDA-10.1.243_GNU-7.3.1_SPMPI-2019.06.24 to build DEBUG code with Kokkos node type CUDA

[rabartl@vortex60 Trilinos (cdofa-102-ats2-fix-run-on-compute-node)]$ echo "'$(atdm_ats2_get_allocated_compute_node_name)'" 
''

[rabartl@vortex60 Trilinos (cdofa-102-ats2-fix-run-on-compute-node)] $ bsub -Is bash
Job <111821> is submitted to default queue <normal>.
<<Waiting for dispatch ...>>
<<Starting on vortex59>>
Don't load the sems modules on 'vortex'!
[rabartl@vortex59 Trilinos]$ echo "'$(atdm_ats2_get_allocated_compute_node_name)'" 
'vortex12'
```

I will add documentation on this function in my next commit.

Reply:
Consider `LSB_HOSTS` changing format such that either an an empty or non-empty string is returned on all nodes.

Reply:
> Consider LSB_HOSTS changing format such that either an an empty or non-empty string is returned on all nodes.

@e10harvey, if that happens, we will update the function `atdm_ats2_get_allocated_compute_node_name` do deal with that.  Currently, that function is only used to get the name of the compute node and set it to `CTEST_SITE` for Experimental builds.  So there is little risk if the format of `LSB_HOSTS` changes.  (And, again, we have a function that encapsulates this.)